### ISSUE TITLE
we need linux test builds

### ISSUE THREAD

**Issue by dfelinto** [dfelinto/blender-multiview-old#24]
@Alexey-Akishin can you upload your builds to graphicall.org?
This would help more people to test the branch.

Many thanks

**Comment by Alexey-Akishin** [comment]
I think I need to add OpenEXR 2 and OIIO to the package somehow, and perhaps build with stable libraries; and test in virtual machine with fresh installation to make sure everything works and there is no missing dependencies. I will try to prepare the package next week...

**Comment by dfelinto** [comment]
Blender usually builds them statically. I can give you some scripts I used to use for Linux building later.

**Comment by Alexey-Akishin** [comment]
Thanks, but I already have building script. Probably I just have to enable WITH_STATIC_LIBS build option to make my package more portable. But I will test my build in clean installation in virtual machine anyway, to be sure (I never build packages for other people before). Hopefully in next few days I will have the package ready.

**Comment by dfelinto** [comment]
ok. windows is building through bulder.blender.org. If you find too complicated I can ask to have Linux set up there as well.

**Comment by Alexey-Akishin** [comment]
My build is ready to be uploaded. I have tested it on Debian Stable, Debian Unstable and Ubuntu LTS. Only thing that I'm missing is an account at blenderstorm so I can log in to graphicall. I have requested it and waiting almost 30 hours now, but after talking to someone at their IRC channel I think it may take a day or two until amino/jesterKing see my requests. As soon as they give me an account, I will upload my build.
